home-work/styles.css

```diff
@@ -0,0 +1,51 @@
+
+.first-block{
+    width: 70%;
```

Comment:
Hi Priya, since you have an `inline-block` class, you don't need this `display` property here. You could simple use the `inline-block` class on those elements